doc/guides/bluetooth/gap-pics.rst

```diff
@@ -35,8 +35,9 @@ Parameter Name	Selected	Description
 TSPC_GAP_0A_1	False (*)	Core Specification Addendum 3 (CSA3) (C.1)
 TSPC_GAP_0A_2	False (*)	Core Specification Addendum 4 (CSA4) (C.2)
 TSPC_GAP_0A_3	False (*)	Core Spec version 4.1 (Core v4.1) (C.3)
-TSPC_GAP_0A_4	True		Core Spec version 4.2 (Core v4.2) (C.4)
-TSPC_GAP_0A_5	False (*)	Core Spec version 5.0 (Core v5.0) (C.4)
+TSPC_GAP_0A_4	False (*)	Core Spec version 4.2 (Core v4.2) (C.4)
+TSPC_GAP_0A_5	True		Core Spec version 5.0 (Core v5.0) (C.5)
+TSPC_GAP_0A_6	False (*)	Core Spec version 5.1 (Core v5.1) (C.6)
```

Comment:
I think the consensus was that we'd do a 5.1 listing.

Reply:
FYI, Change to 5.1 will:
GAP
Remove:
```
GAP/CONN/UCON/BV-06-C
GAP/PRIV/CONN/BV-10-C
GAP/PRIV/CONN/BV-11-C
GAP/SEC/AUT/BV-17-C
GAP/SEC/AUT/BV-18-C
GAP/SEC/AUT/BV-19-C
GAP/SEC/AUT/BV-20-C
GAP/SEC/AUT/BV-21-C
GAP/SEC/AUT/BV-22-C
GAP/SEC/AUT/BV-23-C
GAP/SEC/AUT/BV-24-C
```
Add:
```
GAP/CONN/DCEP/BV-04-C
GAP/SEC/AUT/BV-15-C
GAP/SEC/AUT/BV-16-C
```